llvm/lib/Target/WebAssembly/WebAssemblyInstrSIMD.td

```diff
@@ -165,8 +165,9 @@ def F16x8 : Vec {
  let prefix = "f16x8";
 }
 
-// TODO: Include F16x8 here when half precision is better supported.
-defvar AllVecs = [I8x16, I16x8, I32x4, I64x2, F32x4, F64x2];
+// TODO: Remove StdVecs when the F16x8 works every where StdVecs is used.
```

Comment:
It's not obvious from this patch, but now `AllVecs` is only used in one place for bitcast (which means it now works for f16x8 vectors too).

Alternatively, I can leave `AllVecs` alone and just concat F16x8 down where bitcast is supported.